pkg/cloudprovider/vsphere/route/config/consts_and_errors.go

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package config
+
+import (
+	"time"
+)
+
+const (
+	// ClusterNameTagScope is the scope of clusterName tag
+	ClusterNameTagScope = "vspherecloudprovider/clusterName"
```

Comment:
The Tag is in <key:value> format, scope is the key and node name will be the value. For example if a node name is "node1", the tag will be "vspherecloudprovider/clusterName": "node1". Let me add more discription

Reply:
Thanks, I was specificalyl referring to the tag key `vspherecloudprovider/clusterName`. Wondering if it should be something like `vsphere.k8s.io/cluster-name` or something else. We should be careful about using k8s.io prefixes though since this is technically not an official Kubernetse API.

Reply:
I added `vspherecloudprovider` in the scope because I thought it would be better to add the owner in the tag, so other user could know the static routes are create by it. I'm fine to use `vsphere.k8s.io`.
I checked LB tags, looks like it just use `cluster` as the scope, do you think I should follow the same?

```
const (
	// ScopeOwner is the owner scope
	ScopeOwner = "owner"
	// ScopeCluster is the cluster scope
	ScopeCluster = "cluster"
	// ScopeService is the service scope
	ScopeService = "service"
	// ScopePort is the port scope
	ScopePort = "port"
	// ScopeIPPoolID is the IP pool id scope
	ScopeIPPoolID = "ippoolid"
	// ScopeLBClass is the load balancer class scope
	ScopeLBClass = "lbclass"
)
```